README.md

```diff
@@ -60,7 +60,12 @@ subjects:
   namespace: gitops
 ```
 
-# Making a release
-
-1. Run `make release` to set tag version the multiple required locations.
-2. Create a github release. Set the release tag to match the version number used. The github pipeline will do the following: Create a new version of the chart, create a docker image and release a new python library version to pypi.
+## Contribution
+We used to manually pump versions using `Make` or most recently `mise` but now we're using [releaseplease](https://github.com/googleapis/release-please) to publish a new version do the following:
```

Comment:
Remove mention of the previous release mechanism.

Reply:
done 👍🏼